src/emmetHelper.ts

```diff
@@ -557,6 +565,12 @@ function isExpandedTextNoise(syntax: string, abbreviation: string, expandedText:
 		return false;
 	}
 
+	// users might write successive dots '..', '...' which shouldn't be treated as an abbreviation
+	const successiveDots = abbreviation.match(/^(\.{2,})$/);
```

Comment:
Could this be changed to use `.test()` instead? Because of the comment and the fact that `successiveDots` is only used once, I'm thinking the const can be directly inlined into the if statement as well.
Lastly, are the parentheses needed around the dots?

Reply:
yes you're right, no need to capture them, I will also push a commit to use test and inline it directly as you suggested